too-many-hooks/README.md

```diff
@@ -6,11 +6,11 @@ This directory contains a Vite app which builds to the `too-many-hooks` library
 
 Install / update dependencies with:
 
-```yarn install```
+`yarn install`
```

Comment:
````suggestion
```sh
yarn install
```
````
I think we want to keep these as code blocks, but we do want to add `sh`